compiler/rustc_const_eval/src/interpret/call.rs

```diff
@@ -693,7 +693,7 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
                 trace!("Virtual call dispatches to {fn_inst:#?}");
                 // We can also do the lookup based on `def_id` and `dyn_ty`, and check that that
                 // produces the same result.
-                if cfg!(debug_assertions) {
```

Comment:
why remove this cfg? alternatively, can you move this into a sub-fn, having a dummy block here feels odd.

Reply:
Ralf had suggested above to remove it since it's not too expensive and is a useful check.